react/core/main.tsx

```diff
@@ -223,7 +267,7 @@ function start() {
       // Expose render promise to global context.
       window.rendered = (maybeRenderPromise as Promise<
         NamedServerRendered
-      >).then(({ markups, maxAge, page, renderTimeMetric }) => ({
+      >).then(({ markups, maxAge, page, renderTimeMetric, ...rendered }) => ({
```

Comment:
What are you using from `rendered`?

Reply:
`ampScripts` and `ampHeaderBoilerplate`

Reply:
I find it weird to destructure all properties if I just need to forward them to the return value